docs/content/filtering.md

```diff
@@ -96,191 +134,285 @@ With `--ignore-case`
+* Whenever the `--disable-ListR` flag is applied to an rclone command.
 
-Filtering rules are added with the following command line flags.
+Rclone commands imply directory filter rules from path/file filter
+rules. To view the directory filter rules rclone has implied for a
+command specify the `--dump filters` flag.
 
-### Repeating options ##
+Eg for an include rule
 
-You can repeat the following options to add more than one rule of that
-type.
+    /a/*.jpg
 
-  * `--include`
-  * `--include-from`
-  * `--exclude`
-  * `--exclude-from`
-  * `--filter`
-  * `--filter-from`
-  * `--filter-from-raw`
+Rclone implies the directory include rule
+
+    /a/
+
+Directory filter rules specified in an rclone command can limit
+the scope of an rclone command but path/file filters still have
+to be specified.
 
-**Important** You should not use `--include*` together with `--exclude*`. 
-It may produce different results than you expected. In that case try to use: `--filter*`.
+Eg `rclone ls remote: --include /directory/` will not match any
+files. Because it is an `--include` option the `--exclude **` rule
+is implied, and the `\directory\` pattern serves only to optimise
+access to the remote by ignoring everything outside of that directory.
 
-Note that all the options of the same type are processed together in
-the order above, regardless of what order they were placed on the
-command line.
+Eg `rclone ls remote: --filter-from filter-list.txt` with a file
+`filter-list.txt`:
 
-So all `--include` options are processed first in the order they
-appeared on the command line, then all `--include-from` options etc.
+    - /dir1/
+    - /dir2/
+    + *.pdf
+    - **
 
-To mix up the order includes and excludes, the `--filter` flag can be
-used.
+All files in directories `dir1` or `dir2` or their subdirectories
+are completely excluded from the listing. Only files of suffix
+`'pdf` in the root of `remote:` or its subdirectories are listed.
+The `- **` rule prevents listing of any path/files not previously
+matched by the rules above.
+
+Option `exclude-if-present` creates a directory exclude rule based
+on the presence of a file in a directory and takes precedence over
+other rclone directory filter rules.
 
 ### `--exclude` - Exclude files matching pattern ###
 
-Add a single exclude rule with `--exclude`.
+Excludes path/file names from an rclone command based on a single exclude
+rule. 
 
-This flag can be repeated.  See above for the order the flags are
+This flag can be repeated. See above for the order filter flags are
 processed in.
 
-Eg `--exclude *.bak` to exclude all bak files from the sync.
+`--exclude` should not be used with `--include`, `--include-from`,
+`--filter` or `--filter-from` flags.
 
-### `--exclude-from` - Read exclude patterns from file ###
+`--exclude` has no effect when combined with `--files-from` or
+`--files-from-raw` flags.
 
-Add exclude rules from a file.
+Eg `rclone ls remote: --exclude *.bak` to exclude all .bak files
```

Comment:
This example should have quotes around the `"*.bak"` as on unix at least, if there is file which matches `*.bak` in the current directory it will be substituted here which will suprise users!